src/Cli/dotnet/Commands/BuildServer/Shutdown/BuildServerShutdownCommandDefinition.cs

```diff
@@ -7,8 +7,10 @@
 
 namespace Microsoft.DotNet.Cli.Commands.BuildServer.Shutdown;
 
-internal static class BuildServerShutdownCommandParser
+internal static class BuildServerShutdownCommandDefinition
 {
+    public const string Name = "shutdown";
```

Comment:
I know I've mentioned this before, but wouldn't it make sense to have an interface/type that contains `Name` and `Create()` so that all these Definition classes just implement those? Patterns like this should be reinforced by the type system of the language, not simply by convention.